app/models/service_template_ansible_playbook.rb

```diff
@@ -179,6 +180,11 @@ def retirement_potential?
 
   private
 
+  def retain_resource_actions
+    @resource_actions = resource_actions
+    yield
+  end
```

Comment:
you don't need an instance variable. You can pass them to `delete_job_templates` as a parameter. As the method suggests, passing `job_templates` make more senses as passing `resource_actions`